.github/workflows/pr-checks.yml

```diff
@@ -39,38 +41,45 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v1
+    - name: 'Checkout Source Code'
+      uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      id: buildx
+      uses: docker/setup-buildx-action@v1
 
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+        
     - name: Cache docker layers
-      uses: actions/cache@v1
+      uses: actions/cache@v2
+      id: cache
       with:
-        path: ./caches
-        key: v1-${{ github.head_ref }}
-        
-    - name: Load docker layer cache
-      run: |
-        set +o pipefail
-        docker load -i ./caches/app.tar | true
+        path: /tmp/.buildx-cache
+        key: v2-${{ github.head_ref }}
     
+    - name: 'Docker Prepare'
+      run: docker image prune -a -f
+
     - name: Build devfile registry
-      run: |
-        docker build \
-          --cache-from=app \
-          -t app \
-          -f ./build/dockerfiles/Dockerfile \
-          --target registry .
-           
-    - name: Cache docker layers
-      run: |
-        mkdir -p ./caches
-        docker save -o ./caches/app.tar app
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./build/dockerfiles/Dockerfile
+        target: registry
+        platforms: linux/amd64,linux/s390x
```

Comment:
looks like platforms should be a constant on top of the file to avoid copy/paste

Reply:
@benoitf - Thanks, Raised #325 to address this review comment.